api/src/database/system-data/fields/collections.yaml

```diff
@@ -116,6 +116,24 @@ fields:
       collectionField: collection
     width: full
 
+  - field: branches_divider
+    special:
+      - alias
+      - no-data
+    interface: presentation-divider
+    options:
+      icon: fork_right
+      title: $t:field_options.directus_collections.branches_divider
+    width: full
+
+  - field: branches_enabled
+    interface: boolean
+    special:
+      - cast-boolean
+    options:
+      label: $t:field_options.directus_collections.branches_enabled
+    width: half
+
```

Comment:
I think we should change the divider title to "Content Versioning". Also "Branches Enabled" & "Enabled Branches" feels redundant. So after changing the title, I think we can rename "Branches Enabled" to just "Branches"

<img src="https://github.com/directus/directus/assets/5363448/96147838-6cc6-44a2-89b5-4a80a4d0f1b3">